### PR TITLE
Cerberus 2, rebased

### PR DESCRIPTION
**Pull request by eirnym**

### CODE REVIEW

File: cerberus/tests/test_assorted.py [hunk 1]
```diff
@@ -1,45 +1,10 @@
...
 from decimal import Decimal
-from pkg_resources import Distribution, DistributionNotFound
 
 from pytest import mark
 
-from cerberus import TypeDefinition, Validator
+from cerberus import validator_factory, TypeDefinition, Validator
+from cerberus.base import UnconcernedValidator
 from cerberus.tests import assert_fail, assert_success
-from cerberus.utils import validator_factory
```
Comment (funkyfuture) [inline cerberus/tests/test_assorted.py]: except for tests that target deprecated features, none shall be removed.

Comment (funkyfuture) [reply cerberus/tests/test_assorted.py]: i meant here: https://github.com/pyeve/cerberus/pull/479/files/097fdf3570ab99120d6c6a52a505926a6752f52b#diff-9fea6a85829cee90b14b3752603ce042L23

Comment (eirnym) [reply cerberus/tests/test_assorted.py]: Could you explicitly point each one?

Comment (funkyfuture) [reply cerberus/tests/test_assorted.py]: not too soon. i would go though all changes in the test files and check whether a removal is also present in #431.

Comment (eirnym) [reply cerberus/tests/test_assorted.py]: Thank you for your effort and support

Comment (funkyfuture) [reply cerberus/tests/test_assorted.py]: excuse me, if i'm wrong reading this as 'sarcastic' comment. if i'm not wrong, please consider that all the work here happens in our free time. i appreciate that you took the initiative to update the development here, especially as it requires some effort. i think my last hint is helpful how you could proceed by yourself.

Comment (eirnym) [reply cerberus/tests/test_assorted.py]: I didn't meant any 'sarcastic' meanings here. I'm trying to go the same way as you directed, but it's not an easy thing, as you know by yourself. I asked for a help and know that it requires some effort from both sides to fix and review everything, including documentation examples.

**Comment by funkyfuture** [comment]
Closing in favor of #497.

**Pull request closed without merging**